Frontend/ui-customer/src/Screens/CustomerUtils/TripDetailsScreen/View.purs

```diff
@@ -222,22 +249,17 @@ tripDetailsView state =
           , width WRAP_CONTENT
           , orientation HORIZONTAL
           ][  imageView
-              [ margin if (isJust state.data.vehicleVariant) then (MarginLeft 28) else (MarginLeft 0)
+              [ margin $ MarginLeft $ if isJust state.data.vehicleVariant then 24 else 0
               , cornerRadius 18.0
               -- , background Color.grey800
               , width (V 36)
               , height (V 36)
               , imageWithFallback $ "ny_ic_profile_image," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_profile_image.png"
               ]
             , imageView
-              [ imageWithFallback case (fromMaybe ST.AUTO_RICKSHAW state.data.vehicleVariant) of 
-                                      ST.TAXI -> "ic_sedan_non_ac,"<> (getAssetStoreLink FunctionCall) <>"ic_sedan_non_ac.png"
-                                      ST.TAXI_PLUS -> "ic_sedan_ac,"<> (getAssetStoreLink FunctionCall) <>"ic_sedan_ac.png"
-                                      ST.SEDAN -> "ic_sedan,"<> (getAssetStoreLink FunctionCall) <>"ic_sedan.png"
-                                      ST.SUV -> "ic_suv,"<> (getAssetStoreLink FunctionCall) <>"ic_suv.png"
-                                      ST.HATCHBACK -> "ic_hatchback,"<> (getAssetStoreLink FunctionCall) <>"ic_hatchback.png"
-                                      ST.AUTO_RICKSHAW -> "ic_vehicle_side,"<> (getAssetStoreLink FunctionCall) <>"ic_auto_side_view.png"
-                                      _ -> "ic_sedan_non_ac," <> (getAssetStoreLink FunctionCall) <> "ic_sedan_non_ac.png"
+              [ imageWithFallback $ case state.data.vehicleVariant of
+                                      Just variant -> getVehicleVariantImage (show variant)
+                                      Nothing      -> "ic_vehicle_side,"<> getAssetStoreLink FunctionCall <>"ic_auto_side_view.png" 
```

Comment:
@ashishsingh101 can you verify the asset store image once.

Reply:
verified for YS, others are same as they were